e2e/release/src/release.test.ts

```diff
@@ -122,10 +123,10 @@ describe('nx release', () => {
     const changelogOutput = runCLI(`release changelog 999.9.9`);
     expect(changelogOutput).toMatchInlineSnapshot(`
 
-      >  NX   Generating a CHANGELOG.md entry for v999.9.9
+      >  NX   Generating an entry in CHANGELOG.md for v999.9.9
```

Comment:
I switched round the structure of the log, because now that the path is fully customizable it could end up being reasonably long (depending on the level of nesting within the tree) and for longer paths it reads better this new way round